app/models/embed/purl/resource_file.rb

```diff
@@ -3,21 +3,22 @@
 module Embed
   class Purl
     class ResourceFile
-      # @param [Embed::Purl::Resource] resource
+      # @param [String] druid the identifier of the resource this file belongs to.
+      # @param [String] description the label for this file
```

Comment:
Should we consider calling this `label`?

Reply:
I'll get to that in the next PR.